site/en/docs/react_gemini_prompting.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>Nit: <code>(s)</code> is unnecessary on each section here, each example is singular.</p><p> </p><p>Each ordered list only contains one item too - you can remove the bullets in that case and instead use plain English to introduce. I think you can use the quotes for the first 2 as-is too - just make sure you link back to their source (e.g. the first paragraph should just be something like <code>According to [source], ReAct is a prompting method...</code>, without the <code>What(s)</code> and <code>1. What is ReAct</code>  lead-ins. </p><p> </p><p>It might be worth getting a TW to help review the prose too.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Anirudh is helping me with this already. 
Internal: cl/607799375

will update the cl as well.